example/tests/test_helpers.py

```diff
@@ -46,6 +48,20 @@ def __str__(self):
         return 'Related object'
 
 
+class ArticleProxyModel(Article):
+
+    class Meta:
+        app_label = "articles"
```

Comment:
Use single quotes for strings everywhere please.

Reply:
sorry, a force of habit.